src/parsers/Parse_transform.res

```diff
@@ -4,19 +4,23 @@ type transformProperties = [
   | #translateX
   | #translateY
   | #translateZ
+  | #translate3d
```

Comment:
I added more valid transform values here, although I haven't verified how `renature` actually handles 3D transforms currently. Also removed `skewZ` since it isn't a valid `transform` property!